src/renderers/WebGLRenderer.js

```diff
@@ -1434,6 +1435,7 @@ function WebGLRenderer( parameters ) {
 	function renderObject( object, scene, camera, geometry, material, group ) {
 
 		object.onBeforeRender( _this, scene, camera, geometry, material, group );
+		restoreRenderState( scene, _currentArrayCamera || camera );
```

Comment:
Minor thing... I would just do this:

```js
// restore render state
currentRenderState = renderStates.get( scene, _currentArrayCamera || camera );
```

Same thing after `onAfterRender`.

Reply:
Okay! Done 👍